dcpy/utils/s3.py

```diff
@@ -101,6 +102,22 @@ def client() -> S3Client:
     )
 
 
+def list_buckets() -> list[str]:
+    """Get list of buckets for the defined connection variables"""
+    return [b["Name"] for b in client().list_buckets()["Buckets"]]
+
+
+def get_bucket_region(bucket: str) -> str:
```

Comment:
Very glad that this can be determined programmatically, though see docstring. recipes and publishing are both "nyc3b" while the new dev buckets that get spun up are in "nyc3d"